recipes/cirrus-ci/.cirrus.yml

```
@@ -0,0 +1,5 @@
+contianer:
+  image: mreichelt/android:28
+  
+check_task:
```

How about this:
```suggestion
build_task:
```
Would this more descriptive name work with Cirrus CI?